frontend/app/modals/about.tsx

```
@@ -68,6 +69,18 @@ const AboutModal = ({}: AboutModalProps) => {
     );
 };
 
+AboutModalV.displayName = "AboutModalV";
+
+interface AboutModalProps {}
```

Biome (2.4.4)

[error] An empty interface is equivalent to {}.

(lint/suspicious/noEmptyInterface)